### PR TITLE
Add dotenv and .env-sample to override browser and port configuration…

### PR DESCRIPTION
**Pull request by ibnutri**
… for playwright

## Proposed changes
- add dotenv dependency and .env.test-sample to override default setting for playwright

## Dependencies
- adding dotenv

## Checklist
- [x] Tested locally
- [ ] Added new unit, integration or regression tests
- [x] Updated docs (in confluence)

## Demo
- Add before and after screenshots and videos showcasing the changes

### CODE REVIEW

**Comment by jimee02** [comment]
@ibnutri I think this will break CI/CD.  Better to do this through an npm script like 
```
"test:ui-local": "PLAYWRIGHT_PORT=5000 npx playwright test --ui --config ./tests/playwright/config.js",
```

**Comment by sambua** [comment]
Hi all. 

I think it's good idea not to use variables like port in the scripts as it should be changes each time if user need to run it's in different port, so why `.test.env` implementation will be good idea to make settings persanalized. 

Only as mentioned @jimee02 it will cause trouble on action part. So will be better where we're include `dotenv` check if `process.env.CI` is `false`, by setting CI=false in `.test.env`

So something like: 
```
const CI = process.env?.CI === true || process.env?.CI === "true";

if (!CI) {
  import dotenv from 'dotenv';
  dotenv.config({path:'.test.env'});
}

// after process.env should be applied
```

Or another options rename constants like LOCAL_SERVER_PORT=3333 and use like:

```
const PORT = process.env?.PLAYWRIGHT_PORT || process.env?.LOCAL_SERVER_PORT  || 5000;

```

**Comment by jimee02** [comment]
To add to this, browser path is different on each operating system so a single env will not support all local development

**Comment by sambua** [comment]
Correct so why, Ibnu will add `.test.env.example` as a sample where each user, has to define own path to the executable browser as well parallel to PORT number. After rename this file to `.test.env` which will be git-ignored and setup based per developers machine.

File: package.json [hunk 1]
```diff
@@ -63,7 +63,8 @@
         "sass": "^1.57.1",
         "jsdom-recent": "npm:jsdom@24.0.0",
         "resize-observer-polyfill": "^1.5.1",
-        "nyc": "^15.1.0"
+        "nyc": "^15.1.0",
+        "dotenv": "^16.4.5"
```
Comment (ntshcalleia) [inline package.json:67]: Hi @ibnutri, the contents of `package.json` are overwritten by [package-source.json](https://github.com/betajs/betajs-media-components/blob/master/package-source.json), so we need to update that file too.

**Review by ntshcalleia — CHANGES_REQUESTED** [review]

**Comment by ntshcalleia** [comment]
Hi @ibnutri, we also need to update `package-lock.json` so the action doesn't throw this error:
https://github.com/betajs/betajs-media-components/actions/runs/9318838270/job/25652000552?pr=563#step:3:54

**Pull request closed without merging**